Code/Tools/SceneAPI/SDKWrapper/AssImpSceneWrapper.h

```diff
@@ -43,6 +44,8 @@ namespace AZ
             AZStd::pair<AxisVector, int32_t> GetFrontVectorAndSign() const;
 
             AZStd::string GetSceneFileName() const { return m_sceneFileName; }
+            void CheckTextureName();
+            aiString GetNewTextureName(const aiString& oldName, const aiMesh* pMesh);
```

Comment:
These don't need to be declared in the header, as it looks like its only used by AssImpSceneWrapper.cpp. I'd suggest just making these functions static local functions in that file only so the header here doesnt need to change